Compilation errors

<!-- Thank you for creating this bug report. -->
<!-- Texts like these are comments to help you create the bug report and will not be visible in the final bugreport. -->

**OS**: Fedora release 30 (Thirty)
**Birdtray version**: git upstream
**Thunderbird version**: 68.1.1
**Birdtray origin**: git
**Qt version**: 5.12.5

**Description**
Compilation failure for days using upstream git code

**Expected behavior**
Compiles successfully

**To Reproduce**
<!-- Describe the steps to reproduce the bug. -->
1. (cd src && qmake-qt5 && make)
2. <!-- Step 2 -->
<!-- ... -->

**Screenshots**
<!-- If you have any screenshots to help explain your problem, paste them here. Otherwise you can delete this section. -->

**Additional context**
Initially, I got `error: ‘QCoreApplication’ has not been declared` but then it changed to
```
WARNING: Failure to find: debug/dynamic_de.qm
WARNING: Failure to find: debug/dynamic_en.qm
WARNING: Failure to find: debug/main_de.qm
WARNING: Failure to find: debug/main_en.qm
make: *** No rule to make target 'debug/dynamic_de.qm', needed by 'debug/translations/dynamic_de.qm'.  Stop.
```
git bisect tells me
```
The first bad commit could be any of:
42c5008eb04d9e2754ad1332b03ec2dc2fd8d21d
b0f2f96b5b8f9830de5801c19e39c06319b531c6
bc8ca2592d817f25c2eec9560e7e2bce58f62ec4
c112e6bcb2ab2a247e3eb3c19f7d0974625c5fda
36d0f93dc110ca636e920879a4683409135868a3
6068062884b39629832ba563e4fbf1c5e6ef24e2
1961c3b4615f1c5d1082d86e0881349a62c85555
We cannot bisect more!
```

--- a/src/birdtray.pro
+++ b/src/birdtray.pro
@@ -5,7 +5,7 @@
 #-------------------------------------------------
 
 QT       += core gui network
-CONFIG += c++11 lrelease file_copies
+CONFIG += c++11
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
 TARGET = birdtray
@@ -93,14 +93,22 @@ EXTRA_TRANSLATIONS += \
     translations/dynamic_en.ts \
     translations/dynamic_de.ts
 
-release::OUT_DIR = $$OUT_PWD/release
-debug::OUT_DIR = $$OUT_PWD/debug
-translations.path = $$OUT_DIR/translations
+qtPrepareTool(L_RELEASE, lrelease)
+equals(PWD, $${OUT_PWD}) {
+    QM_FILES_INSTALL_PATH = $$OUT_PWD/translations
+} else { # Qt Creator places the executable in a debug or release directory
+    release::QM_FILES_INSTALL_PATH = $$OUT_PWD/release/translations
+    debug::QM_FILES_INSTALL_PATH = $$OUT_PWD/debug/translations
+}
+!exists($$QM_FILES_INSTALL_PATH) {
+    mkpath($$QM_FILES_INSTALL_PATH) | error("Unable to create directory at $$QM_FILES_INSTALL_PATH")
+}
 for(TRANSLATION_FILE, $$list($$files(translations/*.ts))) {
     TRANSLATION_FILE_NAME = $$basename(TRANSLATION_FILE)
-    translations.files += $$OUT_DIR/$$replace(TRANSLATION_FILE_NAME, .ts, .qm)
+    COMMAND = $$L_RELEASE -removeidentical $$TRANSLATION_FILE \
+                          -qm $$QM_FILES_INSTALL_PATH/$$replace(TRANSLATION_FILE_NAME, .ts, .qm)
+    system($$COMMAND) | error("Failed to generate translations: Failed to run: $$COMMAND")
 }
-COPIES += translations
 
 unix {
      SOURCES += windowtools_x11.cpp

